src/main/app/src/components/Sales.js

```diff
@@ -3,23 +3,33 @@ import { DataGrid } from "@material-ui/data-grid";
 import { useState } from 'react';
 import useSalesPageStyles from "../styles/salesPageStyles";
 import { SpinBeforeLoading } from "./inventory/Inventory";
+import  EditDialog  from "../Forms/EditForm";
```

Comment:
The function you are importing is called `FormDialog` in the original file, but it is imported to a different name (`EditDialog`) in this file.

This import will work just fine, but it is a little bit confusing importing the component to a different name than the function is named in the original file. It works, but it is a bit confusing.